scripts/contributors.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ $1 ]; then
+
+  if [ $2 ]; then
+    HEAD=$2
+  else
+    HEAD=head
+  fi
+  git log '--format=format:* [``%h``](https://github.com/slick/slick/commit/%H) %s' --merges $1..$2 | grep -v "from slick/tmp" | sed -e 's/.* from //' | grep -v "Merge branch" | sed -e 's|/.*||' | sort | uniq | sed -e 's|\(.*\)|, [\1](https://github.com/\1)|'
```

Comment:
I think `$1..$2` should be $1..$HEAD` like in the other scripts.

Otherwise the syntax `./contributors.sh v3.2.3` will not work

Reply:
Correct. Thanks for the spotting one more!

Fixed now.